Tests/FoundationInternationalizationTests/Formatting/NumberFormatStyleTests.swift

```diff
@@ -595,6 +595,39 @@ final class NumberFormatStyleTests: XCTestCase {
         XCTAssertEqual((-922337203685477 as Decimal).formatted(baseStyle.rounded(rule: .awayFromZero, increment: 100)), "-$1000T")
     }
 
+    func testCurrency_Codable() throws {
+        let gbpInUS = Decimal.FormatStyle.Currency(code: "GBP", locale: enUSLocale)
+        let encoded = try JSONEncoder().encode(gbpInUS)
+        let json_gbp = String(data: encoded, encoding: String.Encoding.utf8)!
+        print(json_gbp)
+
+        let previouslyEncoded = """
+{
+    "collection":
+    {
+        "presentation":
+        {
+            "option": 1
+        }
+    },
+    "currencyCode": "GBP",
+    "locale":
+    {
+        "current": 0,
+        "identifier": "en_US"
+    }
+}
+""".data(using: .utf8)
```

Comment:
You should be able to maintain proper indentation here. Any indentation that is present on the closing `"""` won't appear in the actual string.